doc/testing_guide.md

```diff
@@ -76,33 +82,28 @@ yarn start
 ```
 
 ### Local parachain (collator)
-Download the `polkadot-native-storage` parachain repository:
+If you have not done this already, please download and build the polkadot-native-storage parachain repository:
 ```bash
```

Comment:
I think the `please` is a bit over polite here (could be deleted), but tbh, whatever you feel is best.

Reply:
Done.